babel/src/board.js

```diff
@@ -283,20 +280,29 @@ export class Board {
         let hasWideStyleMatch = _.some(_.map(flatChunks, hasMatchInSingleRow));
         return hasWideStyleMatch || _.some(_.map(chunks), hasMatchInPairOfRows);
     }
+    
+    hasMatchEvent() {
+        if (findMatches(this.orbs)[0]) { return true };
```

Comment:
```js
return findMatches(this.orbs)[0];
```

The `return true` part is implied by evaluating the truthiness of `findMatches(this.orbs)[0]`.